test/simulator_tests/cas9_lineage_tracing_simulator_test.py

```diff
@@ -151,15 +155,15 @@ def test_basic_setup(self):
             self.basic_lineage_tracing_data_simulator.mutation_rate_per_character,
         )
 
-        self.assertAlmostEqual(
-            1.0,
-            np.sum(
-                [
-                    v
-                    for v in self.basic_lineage_tracing_data_simulator.mutation_priors.values()
-                ]
-            ),
-        )
+        for i in range(number_of_characters):
+            self.assertAlmostEqual(
+                1.0,
+                np.sum(
+                    [v for v in 
+                    self.basic_lineage_tracing_data_simulator.mutation_priors_per_character[i]
+                    .values()]
+                ),
+            )
 
     def test_setup_errors(self):
```

Comment:
I'd like to see more tests around passing in mutation rates incorrectly (i.e. a list that's too short/not long enough).